# grep appareantly works slightly different on RHEL7 and RHEL8
# see: https://github.com/easybuilders/easybuild-easyconfigs/issues/11394
# SEP 29th 2020 by B. Hajgato (UGent)
diff -ru Multiwfn_3.6_src_Linux.orig/noGUI.sh Multiwfn_3.6_src_Linux/noGUI.sh
--- Multiwfn_3.6_src_Linux.orig/noGUI.sh	2020-09-29 11:50:28.477655541 +0200
+++ Multiwfn_3.6_src_Linux/noGUI.sh	2020-09-29 11:48:33.489854445 +0200
@@ -5,7 +5,7 @@
 cp *.F noGUI
 for f90 in *.f90
 do
-grep -v -E "use dislin_d|use plot|use GUI|call drawdomaingui|call drawmolgui|call drawplanegui|call drawisosurgui|call drawmoltopogui|call drawsurfanalysis|call drawbasinintgui|call drawmol|call drawcurve|call drawscatter|call drawmatcolor|call drawplane|call useclrind|metafl|call window|scrmod|imgfmt|call page|disini|hwfont|axslen|axspos|wintit|call ticks|errmod|labdig|call name|setgrf|call graf|setrgb|call dash|call grid|call solid|call curve|call color|box2d|endgrf|xaxgit|disfin|disini|call ticks|errmod|legini|legtit|call frame|legpos|leglin|legpat|swgfnt|setscr|symfil|messag|imgfmt|complx|sursze|call center|rlmess|call rline|call height|setxid|winsiz|autres|ax3len|crvmat|getscr|ticpos|call labels|call dot|winsiz|namdis|call linwid|call legend|call setboxgui|call selfilegui|call selcolor|call setcolor|call minigui|call texmod|call triplx|call legopt|call myline|call hname|call setgraphformat" $f90 -i > noGUI/$f90
+LC_ALL=C grep -v -E "use dislin_d|use plot|use GUI|call drawdomaingui|call drawmolgui|call drawplanegui|call drawisosurgui|call drawmoltopogui|call drawsurfanalysis|call drawbasinintgui|call drawmol|call drawcurve|call drawscatter|call drawmatcolor|call drawplane|call useclrind|metafl|call window|scrmod|imgfmt|call page|disini|hwfont|axslen|axspos|wintit|call ticks|errmod|labdig|call name|setgrf|call graf|setrgb|call dash|call grid|call solid|call curve|call color|box2d|endgrf|xaxgit|disfin|disini|call ticks|errmod|legini|legtit|call frame|legpos|leglin|legpat|swgfnt|setscr|symfil|messag|imgfmt|complx|sursze|call center|rlmess|call rline|call height|setxid|winsiz|autres|ax3len|crvmat|getscr|ticpos|call labels|call dot|winsiz|namdis|call linwid|call legend|call setboxgui|call selfilegui|call selcolor|call setcolor|call minigui|call texmod|call triplx|call legopt|call myline|call hname|call setgraphformat" $f90 -i > noGUI/$f90
 done
 echo "Now you should run:"
 echo "cd noGUI"
